Vagrantfile

```diff
@@ -0,0 +1,62 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV['VAGRANT_DEFAULT_PROVIDER'] ||= "virtualbox"
```

Comment:
:-1:  I like VMware cos it's faster. Why have this and not let Vagrant use what ever it determines is best?